tests/utils/subnet.go

```diff
@@ -110,14 +110,10 @@ func GetDefaultChainURI(blockchainID string) string {
 // 2. Hardhat contract environment is located at ./contracts
 // 3. Hardhat test file is located at ./contracts/test/<test>.ts
 // 4. npx is available in the ./contracts directory
-func RunDefaultHardhatTests(ctx context.Context, test string) {
-	log.Info("Executing HardHat tests on a new blockchain", "test", test)
-
-	genesisFilePath := fmt.Sprintf("./tests/precompile/genesis/%s.json", test)
-
-	blockchainID := CreateNewSubnet(ctx, genesisFilePath)
+func RunDefaultHardhatTests(ctx context.Context, blockchainID string, test string) {
```

Comment:
could we update the comment here to remove the assumptions that no longer apply?